src/rules/converters/interface-name.ts

```diff
@@ -4,7 +4,7 @@ export const convertInterfaceName: RuleConverter = () => {
     return {
         rules: [
             {
-                ruleName: "@typescript-eslint/interface-name-prefix",
+                ruleName: "@typescript-eslint/naming-convention",
```

Comment:
Similar to `class-name`, [TSLint's `interface-name`](https://palantir.github.io/tslint/rules/interface-name/) specifies a default behavior -- and I'm thinking this converter is actually already buggy (at least per docs, if not in reality) because `interface-name`'s docs says it requires one of either `"always-prefix"` or "`never-prefix`".

Reply:
Same as above comment, I will take a look this weekend :)

Reply:
This one is easy, there is an example at the end of the docs: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-interface-names-do-not-begin-with-an-i

Reply:
Well, sweet.